### DISCUSSION TITLE
Possible to run multiple instances of TFT on the same PC with the bot controlling all of them?

### DISCUSSION THREAD

**Discussion by StereoSayan** [TeamFightTacticsBots/Alune#67] [Q&A] [answered]
Amazing bot that helped me farm alot of tokens from previous battlepasses. Big thanks to all the contributors.
Is it possible to have multiple different TFT instances running each with their separate LoL account on one PC and have the bot control all of them?
Also I saw someone mention different compositions, can you have the bot use different compositions?
What about having it run different gamemodes? (excluding ranked)
What would be the best setup for it to farm the actual LoL event battle pass tokens?

**Comment by akshualy** [comment, marked as answer]
> Is it possible to have multiple different TFT instances running each with their separate LoL account on one PC and have the bot control all of them?

You can run Alune multiple times, I'm currently re-adding the configuration option to manually override which port it's trying to connect to.

> Also I saw someone mention different compositions, can you have the bot use different compositions?

Yes, you can see all supported traits here: https://github.com/TeamFightTacticsBots/Alune/tree/main/alune/images/traits. They can be changed in the config.

> What about having it run different gamemodes? (excluding ranked)

At the moment only normal mode and the new heroes of dawn mode are supported.

> What would be the best setup for it to farm the actual LoL event battle pass tokens?

Just have it play out games, for the event battle passes play time is important.

**Reply by StereoSayan** [reply]
So its possible to run multiple bot instances right now? How would I go about doing that? Do each of them have to be connected to separate ports?

**Reply by akshualy** [reply]
Yes it is possible, if you have multiple Alune folders. Right now you will run into the issue that all of them do port discovery and might try to connect to the same emulator, but I will be fixing that today.
